src/components/Charts/ChartVaccinations.tsx

```diff
@@ -209,17 +215,32 @@ const VaccinationLines: React.FC<{
           </Styles.Axis>
           <RectClipGroup width={innerWidth} height={innerHeight + 2}>
             {seriesList.map(({ label, data, type, params }) => (
-              <ChartSeries
-                key={`series-chart-${label}`}
-                data={data}
-                x={getXPosition}
-                y={getYPosition}
-                type={type}
-                yMax={innerHeight}
-                barWidth={0}
-                barOpacity={1}
-                params={params}
-              />
+              /* Create separate lines for capped and un-capped data points.
+              Capped data will have a dashed line */
+              <Group key={`series-chart-${label}`}>
```

Comment:
using `yarn start` displayed console warnings that `yarn serve` didn't. They were relating to unique keys of the elements. 

![image](https://user-images.githubusercontent.com/55333380/148861562-da4895fe-d461-41e4-9fd9-90ef053b1ce0.png)

There are some other warnings but those seem to also exist on `develop`